pkg/operator/ceph/cluster/mgr/mgr.go

```diff
@@ -169,7 +169,8 @@ func (c *Cluster) Start() error {
 	// If the mgr is newly created, wait for it to start before continuing with the service and
 	// module configuration
 	for _, d := range deploymentsToWaitFor {
```

Comment:
We no longer need this loop since the pod query for the mgr label will naturally query all the mgrs. How about moving lines 172-175 up inside the `if` condition around line 166?

Reply:
moved the code to the if clause